eval/run_generated_agent_evaluation.py

```diff
@@ -41,23 +42,41 @@ def run_evaluation(
 
         if eval_result.checkpoint_results:
             print("Checkpoint results:")
+            obtained_score = 0
+            max_score = 0
             for i, cp_result in enumerate(eval_result.checkpoint_results):
                 print(f"\tCheckpoint {i + 1}:")
                 print(f"\t\tCriteria: {cp_result.criteria}")
                 print(f"\t\tCriteria Points: {cp_result.points}")
                 print(f"\t\tPassed: {cp_result.passed}")
                 print(f"\t\tReason: {cp_result.reason}")
+                obtained_score += cp_result.points if cp_result.passed else 0
+                max_score += cp_result.points
+
+            # Save evaluation results to a JSON file
+            checkpoint_results = eval_result.model_dump()["checkpoint_results"]
+            eval_result_dict = {
+                "obtained_score": obtained_score,
+                "max_score": max_score,
+                "checkpoint_results": checkpoint_results,
+            }
+            with Path("generated_workflows/latest/evaluation_results.json").open("w", encoding="utf-8") as f:
+                f.write(json.dumps(eval_result_dict, indent=2))
+                print("Successfully saved evaluation results to: generated_workflows/latest/evaluation_results.json")
```

Comment:
anyway to make this configurable as a param to the script so that it can support generating them for archived agents aswell?